website/docs/files.md

```diff
@@ -50,3 +50,4 @@ filter "system:MacOSX"
 * [Adding Source Files](Adding-Source-Files.md)
 * [Removing Values](Removing-Values.md)
 * [vpaths](vpaths.md)
+* [includedirs](https://premake.github.io/docs/includedirs/) (Whereas `files` is used to determine which files are included in a generated project's view of what files are associated with that project, `includedirs` is used to determine *where to automatically search for source files* when including/importing them. Being aware of this distinction is important if you want to use a non-standard project folder structure.) 
```

Comment:
> `files` is used to determine which files are included in a generated project's view of what files are associated with that project

I don't understand this sentence.  Generally, `files` is used to determine which c/cpp files **will be compiled and then linked** into a binary. From a compilation point of view, it has nothing to do with "project's view" or header files. I know that you can use this option to "add" header files to see them in Visual Studio project's view, but please remember that Premake is not only for generating Visual Studio projects.

In CMake you have this distinction as well:

```cmake
add_executable(my_binary
    src/file1.cpp
    src/file2.cpp
)

target_include_directories(my_binary
    include/
    vendor/include/
)
```

If you look at the command that is executed during the compilation of your project (msvc, gcc, clang) you will see that _source files_ and _include directories_ are separate parameters passed to the compiler.

I would even say that "_being aware of this distinction_" is not only important but also crucial to survive in C/C++ world 😄